source/client/options_impl.cc

```diff
@@ -373,6 +402,13 @@ OptionsImpl::OptionsImpl(const nighthawk::client::CommandLineOptions& options) {
   trace_ = PROTOBUF_GET_WRAPPED_OR_DEFAULT(options, trace, trace_);
   open_loop_ = PROTOBUF_GET_WRAPPED_OR_DEFAULT(options, open_loop, open_loop_);
   tls_context_.MergeFrom(options.tls_context());
+
+  if (options.has_transport_socket()) {
+    envoy::api::v2::core::TransportSocket ts;
+    ts.MergeFrom(options.transport_socket());
+    transport_socket_.emplace(ts);
```

Comment:
(optional) What is the reasoning behind using .emplace() here? "ts" is an lvalue so it might not necessary, we could as well write something like this:

transport_socket_ = ts;

Which might paint a cleaner picture for the reader.

Reply:
Replaced with a clearer usage of emplace.

Reply:
Done